lang/scala/scala.py

```diff
@@ -0,0 +1,284 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+mode: user.scala
```

Comment:
We've migrated to just using a tag for these matches as part of #776. Please update when you have a moment.

see https://github.com/knausj85/knausj_talon/blob/2d9f4afd73c2566fd5d3bc4a89251c746245cdbe/lang/csharp/csharp.py#L5

and

https://github.com/knausj85/knausj_talon/blob/2d9f4afd73c2566fd5d3bc4a89251c746245cdbe/lang/csharp/csharp.talon#L1

Reply:
I've done this now! Just checking: I don't have to define the tag anywhere, just use it in the header of the talon and py file? Magic!